### PR TITLE
Added ITQ to ADOPTERS.md

### PR DESCRIPTION
**Pull request by mestredelpino**
## What this PR does / why we need it

We were asked by @pnbrown to add ITQ to the adopters page.

## Details for the Release Notes (PLEASE PROVIDE)

```release-note

```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Comment by stmcginnis** [comment]
Awesome, excited to get this added. Looks like we need the CLA signed first. Once signed, I think the CLA bot will report back and we can get this merged.

**Comment by pnbrown** [comment]
Thanks for submitting Carlos!

Could we get this more like the Carvel format of logo at the top and description at the bottom?
https://github.com/vmware-tanzu/carvel/blob/develop/ADOPTERS.md

File: ADOPTERS.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ If you're using Tanzu Community Edition and want to add your organization to thi
...
 
+![ITQ Logo](logos/ITQ.png)(https://itq.eu.com/)
+
 ## Solutions built with Tanzu Community Edition
 
 Below is a list of solutions where Tanzu Community Edition is being used as a component:
+
+### [ITQ Consultancy](https://itq.eu/)
+
+ITQ Consultancy is building up an array of Kubernetes clusters with Tanzu Community Edition, in order to provide their clients (and potential clients) with an environment in which they can develop a Proof of Concept of their choice. This will be provisioned by using the script mentioned on [this blogpost](https://www.mestredelpino.com/automated-tanzu-community-edition-deployment-on-vmware-cloud-on-aws)
```
Comment (jpmcb) [inline ADOPTERS.md:15]:
```
ADOPTERS.md:15:405 MD047/single-trailing-newline Files should end with a single newline character
```

Markdown linter here failing with needing a newline at the end

**Comment by joshrosso** [comment]
> @mestredelpino, VMware has rejected your signed contributor license agreement. The merge can not proceed until the agreement has been resigned. Click [here](https://cla.vmware.com/cla/44a94029424b4e518dc4d58359e7515f) to resign the agreement.

@mestredelpino, I went ahead and updated this PR to fix the small linter issues.

Would you be able to sign the CLA (maybe re-sign) it so we can merge this in.

Sorry for the inconvenience; we really appreciate the contribution.

**Review by joshrosso — APPROVED** [review]

**Review by stmcginnis — APPROVED** [review]

**Pull request merged**